src/components/ArticleList.jsx

```diff
@@ -15,10 +16,12 @@ export default function ArticleList({ articles }) {
   return (
     <div style={style}>
       {articles.map(({ article_id, title, topic, article_img_url }) => {
+        const urlFriendlyTitle = title.replaceAll(/[^a-z]/gi,'-')
+        const articleUrl = '/articles/' + urlFriendlyTitle + '-' + article_id
         return (
           <ArticleCard key={article_id}>
             <div style={childStyle}>
-              <h3>{title}</h3>
+              <Link to ={articleUrl}><h3>{title}</h3></Link>
```

Comment:
Is this necessary ?
could you just link to {`/articles/article.article_id }

Reply:
Not necessary, but makes the URLs human readable for social sharing.